spec/services/pieces_justificatives_service_spec.rb

```diff
@@ -19,9 +19,8 @@
       expect(subject.any? { |piece| piece.name == 'piece_justificative_file' }).to be_falsy
     end
 
-    it 'returns export pdf of the dossier' do
-      expect(champ_identite.piece_justificative_file).to be_attached
-      expect(subject.any? { |piece| piece.name == 'pdf_export_for_instructeur' }).to be_truthy
+    it "doesn't return export pdf of the dossier" do
+      expect(subject.any? { |piece| piece.name == 'pdf_export_for_instructeur' }).to be_falsy
```

Comment:
P-ê que d'écrire un test qui vérifie que le fichier n'est pas présent, on pourrait plutôt déplacer ce test dans celui qui teste `create_list_from_dossier`, pour vérifier que l'export du dossier est bien inclus ?

Reply:
Oui, effectivement, cela me parait plus pertinent.

Reply:
Ce test "it doens't return export pdf of the dossier" est davantage pour indiquer explicitement à la personne qui le code ce que renvoie et ne renvoie pas la méthode